units/src/amount/signed.rs

```diff
@@ -168,7 +222,22 @@ impl SignedAmount {
         SignedAmount::from_str_in(&value.to_string(), denom)
     }
 
-    /// Constructs a new object that implements [`fmt::Display`] using specified denomination.
+    /// Constructs a new object that implements [`fmt::Display`] in the given [`Denomination`].
+    ///
+    /// This function is useful if you do not wish to allocate. See also
+    /// `SignedAmount::to_string_in`.
```

Comment:
This could do with a link. While checking the column width I realised that perhaps because `Amount` and `SignedAmount` have the same API it would be kind of pleasant on the eye to use '[\`Self::foo\`]' in both modules instead of '[\`SignedAmount::foo\`]' - what do you think?

Reply:
I have changed it to ``[`Self::foo`]`` in signed and unsigned, I agree it is better and there were already a couple of cases in the rustdocs.